introduction.md

```diff
@@ -0,0 +1 @@
+This is my introduction. My name is Rachel. I'm learning how to use GitHub right now!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.